modules/gapi/test/streaming/gapi_streaming_vpl_core_test.cpp

```diff
@@ -444,6 +509,182 @@ TEST(OneVPL_Source_ProcessingEngine, Init)
     cv::gapi::wip::Data frame;
     engine.get_frame(frame);
 }
+
+#ifdef HAVE_DIRECTX
+#ifdef HAVE_D3D11
+TEST(OneVPL_Source_DX11_Accel, Init)
+{
+    using namespace cv::gapi::wip::onevpl;
+
+    std::vector<CfgParam> cfg_params_w_dx11;
+    cfg_params_w_dx11.push_back(CfgParam::create<uint32_t>(CfgParam::acceleration_mode(),
+                                                           MFX_ACCEL_MODE_VIA_D3D11));
+    VPLDX11AccelerationPolicy accel(std::make_shared<CfgParamDeviceSelector>(cfg_params_w_dx11));
+
+    mfxLoader mfx_handle = MFXLoad();
+
+    mfxConfig cfg_inst_0 = MFXCreateConfig(mfx_handle);
+    EXPECT_TRUE(cfg_inst_0);
+    mfxVariant mfx_param_0;
+    mfx_param_0.Type = MFX_VARIANT_TYPE_U32;
+    mfx_param_0.Data.U32 = MFX_IMPL_TYPE_HARDWARE;
+    EXPECT_EQ(MFXSetConfigFilterProperty(cfg_inst_0,(mfxU8 *)CfgParam::implementation(),
```

Comment:
`(mfxU8 *)`- c-style cast. I think all c-style casts should be changed to ++ casts.